doc/classes/Callable.xml

```diff
@@ -203,7 +203,8 @@
 		<method name="is_null" qualifiers="const">
 			<return type="bool" />
 			<description>
-				Returns [code]true[/code] if this [Callable] has no target to call the method on.
+				Returns [code]true[/code] if this [Callable] has no object to call the method on and the method is empty. Equivalent to [code]callable == Callable()[/code].
```

Comment:
Callable can be either "object + name" or "custom", after this change it only describes the former case.

I think the previous first sentence is okay. The main issue of https://github.com/godotengine/godot-docs/issues/9831 is the wrong use of "valid" in the description.

Reply:
It checks against the `object` pointer and ignores any special treatment for custom, but can restore to `target`

Reply:
I think "method is empty" is also unnecessary as "target" includes the combination of object and name.

Reply:
If using target yes, meant to restore to the original wording (with the addition of the `==` part)